### ISSUE TITLE
vampire shapeshifting doesn't work

### ISSUE THREAD

**Issue by ATH1909** [tgstation/tgstation#53516]
steps to reproduce:
1. be a wizard
2. go to your magic mirror
3. select the vampire race
4. cast shapeshift: bat
5. your character says "squeak", but doesn't actually turn into a bat
6. the spell never recharges
7. relogging while looking at the spells tab may or may not bork things massively and cause runtimes

**Comment by ATH1909** [comment]
```
[00:18:18] Runtime in shapeshift.dm,30: Cannot execute null.Remove().
  proc name: cast (/obj/effect/proc_holder/spell/targeted/shapeshift/cast)
  usr: athath/(Y`shej)
  usr.loc: (Wizard's Den (28,77,1))
  src: Bat Form (/obj/effect/proc_holder/spell/targeted/shapeshift/bat)
  src.loc: null
  call stack:
  Bat Form (/obj/effect/proc_holder/spell/targeted/shapeshift/bat): cast(/list (/list), Y`shej (/mob/living/carbon/human/species/pod))
  Bat Form (/obj/effect/proc_holder/spell/targeted/shapeshift/bat): perform(/list (/list), 1, Y`shej (/mob/living/carbon/human/species/pod))
  Bat Form (/obj/effect/proc_holder/spell/targeted/shapeshift/bat): choose targets(Y`shej (/mob/living/carbon/human/species/pod))
  Bat Form (/obj/effect/proc_holder/spell/targeted/shapeshift/bat): Click()
  Bat Form (/datum/action/spell_action/spell): Trigger()
  Bat Form (/obj/screen/movable/action_button): Click(null, "mapwindow.map", "icon-x=13;icon-y=15;left=1;scr...")
  ATHATH (/client): Click(Bat Form (/obj/screen/movable/action_button), null, "mapwindow.map", "icon-x=13;icon-y=15;left=1;scr...")
Stopped server.
```
Note that, while I had initially spawned this character in as a podperson, I later mirror'd myself into being a luminescent and then a vampire, which makes the references to the podperson race a bit... odd.

I was able to reproduce this twice more, once with another spawned in character, and once with a character I respawned as in the shuttle and then turned into a wizard (to make sure it wasn't a problem with spawning in my bodies using the debug menu).

**Comment by Livrah** [comment]
I got same problem in actual round 2 weeks ago, thought it was becouse of mime, but didnt work after breaking vow of silence too

**Comment by dragomagol** [comment]
Fixed by #54484
